x-pack/plugins/infra/public/containers/waffle/waffle_nodes.gql_query.ts

```diff
@@ -20,6 +20,7 @@ export const waffleNodesQuery = gql`
         nodes(path: $path, metric: $metric) {
           path {
             value
+            displayName
```

Comment:
I wonder whether using `id` and `name` would be more descriptive?

Reply:
`label` would probably be better but I was being consistent with the metadata API.